docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -1263,3 +1006,14 @@ while the other files provide the necessary connection information.
 ====
 For more details on how the discovery of the available properties works, please read the link:https://github.com/k8s-service-bindings/spec#application-projection[Application Projection] part of the Service Binding specification.
 ====
+
+[#configuration-reference]
+== Configuration reference
+
+TIP: Remember that it's possible to quote property names when they contain `.`.
+Indeed, without quoting, Quarkus wouldn't be able to tell where your property ends when parsing the property's "path".
+So if you need to reference a
+`greeting.message` property in one of the configuration files, you can use the following syntax:
+`quarkus.whatever."greeting.message".something=Hello`.
+
+include::{generated-dir}/config/quarkus-kubernetes.adoc[opts=optional,leveloffset=+1]
```

Comment:
Do we have the `quarkus.openshift` and `quarkus.knative` here?

Reply:
As far as I can tell, yes.